### ISSUE TITLE
[Feature Request] Respect --allow-empty flag

### ISSUE THREAD

**Issue by alecgerona** [commitizen-tools/commitizen#247]
## Description
Respect git flag `--allow-empty` when using the cli tool.

## Possible Solution
Allow empty commits using the cli tool.

## Related Issue
Related to #49

**Comment by Kurt-von-Laven** [comment]
Copying the request from linked issue here. It would be ideal to support `--allow-empty-message` as well.

**Comment by Kurt-von-Laven** [comment]
Is #485 sufficiently general that its implementation would additionally address this issue for free?

**Comment by Lee-W** [comment]
I think #485 is solving another issue but could be a workaround of this one. This one https://github.com/commitizen-tools/commitizen/issues/451#issuecomment-1092067170 might potentially be the general solution of this issue

**Comment by Kurt-von-Laven** [comment]
Indeed, being able to pass arbitrary flags to `git commit` seems ideal!
